src/adagrams.js

```diff
@@ -1,15 +1,85 @@
+import { letterPool, scoreChart } from "./letters";
```

Comment:
I like the organization of putting your letter pool and score chart in another file.